disneylandClient/worker.py

```diff
@@ -93,6 +91,15 @@ def run(self):
 
                 self.acquire_cpus(process_name, 1)
 
-                p = Process(name=process_name, target=self.do_job, args=(job,))
+                p = Process(name=process_name, target=self.do_job, args=(job, self.queue))
```

Comment:
I recommend to just pass stub to worker func. It might want to modify job multiple times inside.

Reply:
grpc not able to establish secure call from outside
![screen shot 2017-11-02 at 16 59 25](https://user-images.githubusercontent.com/18559300/32329858-522eb02c-bfef-11e7-930e-d95cffadd178.png)